templates/cisco_asa_show_arp.textfsm

```diff
@@ -0,0 +1,11 @@
+Value Required INTERFACE (\S+)
+Value Required ADDRESS (\d+\.\d+\.\d+\.\d+)
+Value Required AGE (\S+)
+Value Required MAC (\S+)
+
+Start
+  ^\s*${INTERFACE}\s+${ADDRESS}\s+${MAC}\s+${AGE} -> Record
+  # Capture time-stamp if vty line has command time-stamping turned on
```

Comment:
How do you activate the time stamping on ASA? I've tried looking but unable to find the command?

Can you please also add `  ^\s*$$` above the `  ^. -> Error` line?

Also, the Load for and Time source entries, are these the ones we'd see if time stamping was enabled?